docs/WALKTHROUGH.md

```diff
@@ -0,0 +1,394 @@
+# Service Catalog Demonstration Walkthrough
+
+This document outlines the basic features of the service catalog by walking
+through a basic use case.
+
+## Step 0 - Prerequisites
+
+### Starting Kubernetes with DNS
+
```

Comment:
>I worry here that if we give specific instructions for all of the many ways someone could deploy k8s, we'll be here for a while.

We're not aiming for that. Let's do the best we can here, and decide on what we "officially" support later.